README.md

```diff
@@ -43,18 +43,18 @@ const transfer = TransferTransaction.create(
   Deadline.create(),
   Address.createFromPublicKey(
     'C5C55181284607954E56CD46DE85F4F3EF4CC713CC2B95000FA741998558D268',
-    NetworkType.MIJIN_TEST
+    NetworkType.TEST_NET
   ),
   [new Mosaic(new NamespaceId('cat.currency'), UInt64.fromUint(10000000))],
```

Comment:
as I remember `'cat.currency'` was used for MIJIN_TEST.
should we change to `symbol.xym`  ?

Reply:
Yes, good catch @AnthonyLaw. We should keep naming consistent. 

CC @yilmazbahadir.

Reply:
Thanks, updated!